README.md

```diff
@@ -169,6 +169,71 @@ If you want to use `xrpl.js` with React Native you will need to install shims fo
   import './shim'
   ...
   ```
+### Using xrpl.js with Vite React
+
+Similar to above, to get xrpl.js to work with Vite you need to set up a couple aliases in the vite.config.ts file. 
+
+1. If it's a fresh project you can use `npm create vite@latest` then choose the React and TypeScript options.
+
+2. Copy these settings into your `vite.config.ts` file.
+
+```
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import polyfillNode from 'rollup-plugin-polyfill-node'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  define: {
+    'process.env': {}
+  },
+  optimizeDeps: {
+    esbuildOptions: {
+
+        define: {
+          global: 'globalThis',
```

Comment:
this is in our config but predates any changes made for xrpl.js

Reply:
It seems necessary when trying it out in my toy example (a new vite project + basic xrpl.js code to fund a wallet)